paper-submenu.html

```diff
@@ -131,9 +131,9 @@
        * Expand the submenu content.
        */
       open: function() {
-        if (!this.disabled && !this._active) {
+        if (!this.disabled && !this.opened) {
           this.$.collapse.show();
```

Comment:
This line can be removed now; the iron-collapse will respond automatically to `opened` being changed below.